[Documentation] Build

**undefined: handleWBA**
A dependendy is forgotten in the build command.

**To Reproduce**
Steps to reproduce the behavior:
1. Run: `go build ./witch.go ./util.go ./listener.go`
2. Cause: undefined error
```bash
# command-line-arguments
./listener.go:51:9: undefined: handleWBA
./listener.go:75:9: undefined: handleWBA
```

**To resolve it: add ./wba.go dependency**
`go build ./witch.go ./util.go ./wba.go ./listener.go`.

--- a/README.md
+++ b/README.md
@@ -11,7 +11,7 @@ Windows / Mac / Linux
 ```bash
 cd witch
 go get -u github.com/gen2brain/dlgs
-go build ./witch.go ./util.go ./listener.go
+go build ./witch.go ./util.go ./wba.go ./listener.go
 ```
 ## New
 # Witch DevServer

